src/app/components/EmailNotificationSettingsDialog/components/Subscriptions.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+
+import { translations } from '../../../../locales/i18n';
+import { useEmailNotificationSettingsContext } from '../contexts/EmailNotificationSettingsContext';
+import { useHandleSubscriptions } from '../hooks/useHandleSubscriptions';
+import { Switch } from '@blueprintjs/core/lib/esm/components/forms/controls';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+type SubscriptionsProps = {
+  isDisabled?: boolean;
+};
+
+export const Subscriptions: FC<SubscriptionsProps> = ({ isDisabled }) => {
+  const { t } = useTranslation();
+  const {
+    marginCallsToggle,
+    spotCallsToggle,
+  } = useEmailNotificationSettingsContext();
+  const {
+    marginCallsToggleHandler,
+    spotCallsToggleHandler,
+  } = useHandleSubscriptions();
+
+  return (
+    <div className="tw-my-6">
+      <Switch
```

Comment:
These 2 switches are completely identical - you should at least extract their `className` to a constant.